app/main.py

```diff
@@ -1,3 +1,56 @@
+from __future__ import annotations
+
+
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> Distance:
+        return Distance(round((self.km / other), 2))
+
+    def __lt__(self, other: Distance | int | float) -> bool:
+        return (self.km < other.km
+                if isinstance(other, Distance)
+                else self.km < other)
+
+    def __gt__(self, other: Distance | int | float) -> bool:
+        return (self.km > other.km
+                if isinstance(other, Distance)
+                else self.km > other)
+
+    def __eq__(self, other: Distance | int | float) -> bool:
+        return (self.km == other.km
+                if isinstance(other, Distance)
+                else self.km == other)
+
+    def __le__(self, other: Distance | int | float) -> bool:
+        return (self.km <= other.km
+                if isinstance(other, Distance)
+                else self.km <= other)
+
+    def __ge__(self, other: Distance | int | float) -> bool:
+        return (self.km >= other.km
```

Comment:
Imho this way code will be more readable 
![image](https://github.com/mate-academy/py-distance-class/assets/102595649/5c040220-b2c0-4520-93e1-4c81a4243749)